api/posts.json

```
@@ -0,0 +1,16 @@
+---
+layout: none
+---
+[
+  {%- for post in site.posts -%}
+    {
+      "title":      {{- post.title | jsonify }},
+      "url":        {{- post.url | relative_url | jsonify }},
+      "date":       {{- post.date | date: date_format }},
+      "tags": [
+        {{post.categories | jsonify}}
+      ],
```

```suggestion
      "tags": {{post.categories | jsonify}}
```

Não conheço muito de Jekyll e nem como usar a api dele, mas vi na doc que as `categories` já retornam um array. Então não precisa passar esse `[ ]` ali.

---

Você está certo, vou corrigir